cmd/exoip/main.go

```diff
@@ -258,15 +259,42 @@ func main() {
 		os.Exit(0)
 	}
 
+	if (*address)[0] == ':' {
+		interfaces, err := net.Interfaces()
+		if err == nil {
+		outterfor:
+			for _, iface := range interfaces {
+				if iface.Flags&net.FlagLoopback != 0 || iface.Flags&net.FlagUp == 0 {
+					continue
+				}
+
+				addrs, err := iface.Addrs()
+				if err != nil {
+					continue
+				}
+				for _, addr := range addrs {
+					ip, _, err := net.ParseCIDR(addr.String())
+					if err != nil {
+						continue
+					}
+					if ip != nil && ip.To4() != nil {
+						*address = ip.String()
+						break outterfor
+					}
+				}
+			}
+		}
+	}
```

Comment:
@JessicaToscani this little dance should put the first non-loopback IPv4 address as default value for `IF_BIND_TO`